template/en/default/global/header.html.tmpl

```diff
@@ -120,6 +120,7 @@
             },
             constant => {
                 COMMENT_COLS => constants.COMMENT_COLS,
+                URL_BASE => urlbase,
```

Comment:
I think you can avoid adding this and use BUGZILLA.config.basepath

Reply:
This was used for the smartlinking stuff, which was removed anyway.